robottelo/constants.py

```diff
@@ -291,14 +291,24 @@
 DEFAULT_ARCHITECTURE = 'x86_64'
 DEFAULT_RELEASE_VERSION = '6Server'
 
-LANGUAGES = [
-    u'zh_TW',
-    u'Galego',
-    u'Deutsch',
-    u'it',
-    u'日本語',
-    u'sv_SE',
-]
+LANGUAGES = {
```

Comment:
Catch a bug:

with the change from list to dict the following line will break https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/ui/test_user.py#L572

should be changed to `random.choice(LANGUAGES.keys())`